### PR TITLE
Fix circular dependency

### PR DESCRIPTION
**Pull request by jbalboni**
This is causing errors in prod right now: http://sentry.vetsgov-internal/vets-gov/website-production/issues/17609/

login-helpers imports login actions, which imports login-helpers.

### CODE REVIEW

**Review by cehsu — APPROVED** [review]

**Review by bshyong — APPROVED** [review]
LGTM

File: src/js/login/actions/index.js [hunk 1]
```diff
@@ -62,9 +69,3 @@ export function toggleSearchHelpUserMenu(menu, isOpen) {
   };
 }
 
-export function toggleLoginModal(isOpen) {
```
Comment (cehsu) [inline src/js/login/actions/index.js]: Curious why this got moved (not blocking)?

**Review by U-DON — APPROVED** [review]
LGTM.

**Comment by omgitsbillryan** [comment]
LGTM.  I had originally put the GA client ID stuff in `login-helpers` since it was login'ish.  I have no problem moving it to `'helpers` instead.

On a side note, the sentry issue you linked to is about a `TypeError`
```
n.props.toggleLoginModal is not a function
```
you had to dig further to find the circular dependency?

**Review by omgitsbillryan — APPROVED** [review]

**Comment by jbalboni** [comment]
@omgitsbillryan Yeah, it totally makes sense to put it in login-helpers, it just happens to create the circular dependency.

Yeah, so the problem was manifesting as everything imported from login/actions in my FormPage component being undefined. I spent 45 minutes tearing my hair out trying to find the reason and started commenting things out in login/actions and then in login-helpers until it magically started working again.

There must be some webpack weirdness around when it manifests, since it definitely doesn't seem to be happening everywhere.

**Pull request merged**